org.srb2.SRB2Persona.yaml

```diff
@@ -0,0 +1,104 @@
+app-id: org.srb2.SRB2Persona
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: srb2persona
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
Honestly, I have no idea. I just kept this permission to avoid breaking something, since SRB2's Flatpak has it. Should I remove?